lib/language.js

```diff
@@ -5244,6 +5254,7 @@ function init() {
     ,'Remove %1 selected records?' : {
       cs: 'Odstranit %1 vybraných záznamů'
       ,nb: 'Fjern %1 valgte elementer?'
+      ;de: 'Ausgewählte %1 Aufnahmen löschen?'
       ,ro: 'Șterg %1 înregistrări selectate?'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2470436128/issues/source?bid=2452379&fileBranchId=3259755#l5258)